packages/dso-toolkit/src/variables/states.scss

```diff
@@ -3,18 +3,22 @@
 $success-text: colors.$zwart;
 $success-bg: colors.$success-bg-color;
 $success-border: colors.$success-bg-color;
+$success-border-accentuated: colors.$success-color;
```

Comment:
Andere naam bedenken

Reply:
Gedaan

Reply:
Zie Slack, ik had verwacht dat er ook iets met "accentuated" zou worden gedaan. Dit is een term die we niet voeren, en toepassen om een variant "compact".

Reply:
In `states.scss`:  `$success-compact-accent-color: colors.$success-color;`

In `alert.variables.scss` : `$success-compact-accent-bg: states.$success-compact-accent-color;`